scheme/nanopass-scheme/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                nanopass-scheme
+categories          scheme
```

Comment:
This introduces a new category. Do we expect to get more scheme ports in the future? If not, I would just put it into one of the existing categories (maybe "lang"?).